content/md/assets/assets-transformation.md

```diff
@@ -18,7 +18,7 @@ Say we have a `packshots` asset family. In its structure, it has 2 media file at
 
 The PIM can automatically generate the thumbnail version of your main image for you, and this, thanks to the definition of a transformation!
 
-![Assets Transformations](../img/Assets_Transformations.png)
+![Assets Transformations](../img/Assets_TransformationsV2.png)
```

Comment:
We use the same image twice on the same page?

Reply:
yes, it's normal. But if it's weird, I can remove it.